Wikimate.php

```diff
@@ -62,6 +62,13 @@ public function login( $username, $password ) {
 			'lgpassword' => $password,
 			'format' => 'json'
 		);
+
+		// If $domain is informed, sets the correspondant detail in the request information array
+		if( $domain )
```

Comment:
Should this be a more robust check, e.g. `$domain !== NULL` or `is_string($domain)`?